spp_mis_demo/__manifest__.py

```diff
@@ -28,6 +28,7 @@
         "g2p_programs",
         "product",
         "stock",
+        "spp_custom_filter_ui",
```

Comment:
Same here @shashikala1998, add the `spp_custom_filter` not the `spp_custom_filter_ui`.